api/CanvasGradient.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "CanvasGradient": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasGradient",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": "2.1"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "3.6",
+            "notes": "Before Gecko 5.0, (Firefox 5.0 / Thunderbird 5.0 / SeaMonkey 2.2), specifying non-finite values when adding color stops through a call to <code>addColorStop()</code> incorrectly throws <code>SYNTAX_ERR</code> instead of <code>INDEX_SIZE_ERR</code>."
```

Comment:
This note should just read "Before Firefox 5, specifying non-finite (...)".
We're not mentioning Gecko, Thunderbird and SeaMonkey anymore.

Reply:
OK